st2client/requirements.txt

```diff
@@ -21,8 +21,9 @@ python-dateutil==2.8.1
 python-editor==1.0.4
 pytz==2021.1
 pyyaml==5.4.1
-requests[security]==2.25.1
+requests[security]>=2.27.1,<=2.31.0
 six==1.13.0
 sseclient-py==1.7
 typing-extensions<4.2
+urllib3<2; python_version < '3.7'
```

Comment:
Yeah, environment markers are all buggy with the current pip version 🤯 


Under py3.8 https://app.circleci.com/pipelines/github/StackStorm/st2/4267/workflows/6d50c81f-3524-4f8d-8279-71f47ee5dd67/jobs/16065?invite=true#step-109-925583_94:
```
[package: st2] [20:53:43]      The conflict is caused by:
[package: st2] [20:53:43]          requests[security] 2.31.0 depends on urllib3<3 and >=1.21.1
[package: st2] [20:53:43]          requests 2.31.0 depends on urllib3<3 and >=1.21.1
[package: st2] [20:53:43]          st2client 3.9.dev0 depends on urllib3<2
```
which is wrong.

I can't see any path forward for updating `requests[security]` so far as it requires `urllib3` which is incompatible with python3.6.